sdk/examples/wallet/12_increase_native_token_supply.rs

```diff
@@ -23,6 +23,8 @@ async fn main() -> Result<()> {
 
     // Get the account we generated with `01_create_wallet`
     let account = wallet.get_account("Alice").await?;
+    // May want to ensure the account is synced before sending a transaction.
+    account.sync(None).await?;
 
     let balance = account.balance().await?;
```

Comment:
Syncing also returns the balance, so would be enough here
```suggestion
    let balance = account.sync(None).await?;
```